templates/common/copilot-plugin-existing-api/README.md

```diff
@@ -0,0 +1,43 @@
+# Overview of Copilot Plugin app template
+
+## Build Copilot Plugin from OpenAPI Spec
+
+The plugin allows Copilot to interact directly with third-party data, apps, and services, enhancing its capabilities and broadening its range of capabilities. Plugins allow Copilot to:
+
+- Retrieve real-time information, for example, latest news coverage on a product launch.
+- Retrieve knowledge-based information, for example, my team’s design files in Figma.
+- Perform actions on behalf of the user, for example, create a Jira ticket.
+
+## Get started with Copilot Plugin template
+
+> **Prerequisites**
+>
+> To run the copilot plugin app template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts). Access to Microsoft 365 Copilot is also required.
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-cli)
+
```

Comment:
Join Microsoft 365 Copilot Plugin development [early access program](https://aka.ms/PluginsEarlyAccess).

Reply:
Updated. Also remove "Access to Microsoft 365 Copilot is also required." 
Previously,
![image](https://github.com/OfficeDev/TeamsFx/assets/86260893/cd4d5232-8f90-4e9e-9ff4-60d7530b61f2)